### PR TITLE
Add "constrained" labelAllignment setting

### PR DESCRIPTION
**Pull request by NicholasAntonov**
The constrained labelAllignment setting is similar to center, but hides itself when it cannot fit within the bounds of the node, which leads to a more desirable behavior when zooming out.

### CODE REVIEW

File: plugins/sigma.renderers.linkurious/canvas/sigma.canvas.labels.def.js [hunk 1]
```diff
@@ -26,9 +26,10 @@
         labelWidth,
         labelOffsetX,
         labelOffsetY,
+        shouldRender = true,
         alignment = settings('labelAlignment');
 
-    if (size <= settings('labelThreshold'))
+    if (size < settings('labelThreshold'))
```
Comment (sheymann) [inline plugins/sigma.renderers.linkurious/canvas/sigma.canvas.labels.def.js]: Why this change?

Comment (NicholasAntonov) [inline plugins/sigma.renderers.linkurious/canvas/sigma.canvas.labels.def.js]: I reverted it. It was mistakenly left in after testing.

**Comment by sheymann** [comment]
Looks good, please can you edit the README of the plugin too?

**Comment by NicholasAntonov** [comment]
What would you like edited? The labelAlignment setting is not mentioned in the linkurious readme. It appears to be undocumented.

https://github.com/Linkurious/linkurious.js/search?utf8=%E2%9C%93&q=labelalignment

**Comment by sheymann** [comment]
Oh my! I've documented it on the [wiki](https://github.com/Linkurious/linkurious.js/wiki/Styling) but not in the README. Thanks!

**Comment by sheymann** [comment]
good to go!
